packages/app/src/app/store/modules/files/sequences.js

```diff
@@ -37,6 +37,19 @@ export const createModule = [
   },
 ];
 
+export const uploadFile = [
+  actions.uploadFile,
+  {
+    success: [
+      set(props`newCode`, props`uploadedFile.url`),
+      set(props`title`, props`name`),
+      set(props`isBinary`, true),
+      ...createModule,
+    ],
+    error: [addNotification('Unable to upload file', 'error')],
```

Comment:
I'd use props`error` as message here. I conditioned the API to return nice error messages like 'You've exceeded your maximum storage of 20MB'.